au4/src/projectscene/view/clipsview/au3/au3wavepainter.cpp

```diff
@@ -521,7 +521,11 @@ void DrawMinMaxRMS(int channelIndex, QPainter& painter,
         ColorFromQColor(style.clippedPen))
     .SetEnvelope(clip.GetEnvelope());
 
-    waveformPainter.Draw(channelIndex, painter, paintParameters, metrics);
```

Comment:
actually... why we're dropping metrics passed here before?

Reply:
We adjust taking into account the trim left (just above I take the current metrics and adjust fromTime/toTime)